jax/numpy/__init__.py

```diff
@@ -304,6 +304,7 @@
     ravel_multi_index as ravel_multi_index,
     real as real,
     reciprocal as reciprocal,
+    _register_stackable as _register_stackable,
```

Comment:
My only nit with this PR is that maybe we could consider not including `_register_stackable` in this __init__.py file, and instead importing it differently where it's needed (e.g. instead of `jnp._register_stackable(PRNGKeyArray)` maybe we'd have to do `from jax._src.numpy.lax_numpy import _register_stackable` instead). That'd make extra clear it's an internal API.

Reply:
yep, that makes a lot of sense!